RELEASES.md

```diff
@@ -10,6 +10,8 @@ Please check the [releases](https://github.com/leanprover/lean4/releases) page f
 v4.3.0 (development in progress)
 ---------
 
+* **Lake:** Add `postUpdate?` package configuration option. Used by a package to specify some code which should be run after a successful `lake update` of the package or one of its downstream dependencies. ([lake#185](https://github.com/leanprover/lake/issues/185))
```

Comment:
Do you think it's worth mentioning `lake exe cache get` explicitly here, and noting the mathlib motivation?

Reply:
@eric-wieser I tend to think of a changelog as standalone briefs of *what* was changed and leave the *why* to referenced issues, especially for new features (whereas, e.g., performance improvements, may need to the *why* to understand the significance of the the *what*). However, as the `RELEASES.md` is meant, at least in part, to be a tool for the community. it may be worth further community discussion as to what style of notes would be most helpful.